test/Faker/Provider/BaseTest.php

```diff
@@ -565,6 +566,18 @@ public function testRandomElements(): void
         self::assertCount(3, $allowDuplicates);
         self::assertContainsOnly('string', $allowDuplicates);
     }
+
+    /**
+     * @requires PHP 8.1
+     */
+    public function testRandomElementWithEnum(): void
+    {
+        self::assertCount(2, $cases = BaseProvider::randomElements(BackedEnum::class, 2));
+        self::assertInstanceOf(BackedEnum::class, $cases[0]);
+        self::assertInstanceOf(BackedEnum::class, $cases[1]);
+
+        self::assertInstanceOf(BackedEnum::class, BaseProvider::randomElement(BackedEnum::class));
+    }
```

Comment:
What do you think about

- splitting the tests (one for `randomElements()`, one for `randomElement()`)
- splitting the phases of the tests (arrange, act, and assert)
- using more specific assertions

?

```suggestion
    /**
     * @requires PHP 8.1
     */
    public function testRandomElementsWithEnum(): void
    {
        $count = 2;
        
        $randomElements = BaseProvider::randomElements(BackedEnum::class, $count);

        self::assertCount($count, $randomElements);
        self::assertContainsOnlyInstancesOf(BackedEnum::class, $randomElements);
    }

    /**
     * @requires PHP 8.1
     */
    public function testRandomElementWithEnum(): void
    {
        $randomElement = BaseProvider::randomElement(BackedEnum::class);

        self::assertInstanceOf(BackedEnum::class, $randomElement);
    }
```

Reply:
Makes sense to me 👍